code/modules/reagents/chemistry/reagents.dm

```diff
@@ -233,7 +233,7 @@
 		return
 	if(M.mind && M.mind.changeling && M.mind.changeling.regenerating) //no messing with changeling's fake death
 		return
-	M.emote("deathgasp")
+	M.emote("deathgasp", intentional = FALSE)
```

Comment:
I added some of these but `intentional = FALSE` is the default, so this can be undone